### ISSUE TITLE
Website bugs

### ISSUE THREAD

**Issue by TierneyLiu** [Automattic/VIP-Coding-Standards#627]
Website css file bugs when accessing,  can't event access to wp-admin page with 3 warnings down below

## Bug Description

```php
Warning: Declaration of Ultimatum_Related_Sub_Items_Walker::walk($elements, $max_depth) should be compatible with Walker::walk($elements, $max_depth, ...$args) in /websites/wo/xxxxxx/wp-content/themes/ultimatum/wonderfoundry/helpers/wonderworkshelper.php on line 705

Warning: Cannot modify header information - headers already sent by (output started at /websites/wo/xxxxxx/wp-content/themes/ultimatum/wonderfoundry/helpers/wonderworkshelper.php:705) in /websites/wo/xxxxxxu/wp-includes/pluggable.php on line 1296

Warning: Cannot modify header information - headers already sent by (output started at /websites/wo/xxxxxx/wp-content/themes/ultimatum/wonderfoundry/helpers/wonderworkshelper.php:705) in /websites/wo/xxxxxx/wp-includes/pluggable.php on line 1299
```

**Comment by rebeccahum** [comment]
@TierneyLiu Hi! It looks like you may have stumbled upon the wrong Github issue repo.
